pkg/alerting/drivers/routing/routing.go

```diff
@@ -74,7 +76,7 @@ func NewDefaultOpniRoutingWithOverrideHook(hook string) OpniRouting {
 }
 
 func NewDefaultOpniRouting() OpniRouting {
-	return NewOpniRouterV1("http://localhost:3000")
+	return NewOpniRouterV1(fmt.Sprintf("http://localhost:3000%s", shared.AlertingDefaultHookName))
```

Comment:
Might want to put some belts and braces here to confirm that this constructs a valid URL

Reply:
It eventually translates down to a `url.Url` inside the alertmanager config and is validated at that time, but yeah for posterity we should validate here as well